src/test/play-2/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/addtoalist/addToSummary.scala.html

```diff
@@ -1,38 +0,0 @@
-@import uk.gov.hmrc.hmrcfrontend.views.html.components._
```

Comment:
@matthewmascord Has this example been deleted?

Reply:
Re-instated deleted examples.